.readthedocs.yml

```diff
@@ -2,5 +2,7 @@ requirements_file: dev_tools/conf/pip-list-dev-tools.txt
 build:
   image: latest
 python:
-  version: 3.6
+  version: 3.7
   pip_install: true
+  extras_requirements:
+      - contrib
```

Comment:
If this works, you can also add `dev` here instead of specifying it above.

Reply:
dev would be dev-env now